### ISSUE TITLE
SQL cells should ignore commented interpolated parameters

### ISSUE THREAD

**Issue by nachocab** [observablehq/feedback#584]
On a SQL cell [like this one](https://observablehq.com/@observablehq/sql-cell#filteredQuery), commenting an interpolated parameter should not give an error:

<img width="729" alt="CleanShot 2023-07-28 at 14 18 33@2x" src="https://github.com/observablehq/feedback/assets/27538/4a45e7e6-6826-4924-aa82-67551b708d92">

**Comment by mootari** [comment]
I don't think we can prevent that, as it would require parsing the query syntax to determine where the interpolation happens.

**Comment by nachocab** [comment]
I understand, but remember that comments in SQL are easier than in JS. It would be as simple as running this regex: `query.replace(/--.+/g, '')`

**Comment by mootari** [comment]
No, multiline comments (`/* ... */`) are also supported.

**Comment by nachocab** [comment]
Oh, I didn't know that. Anyway, handling single line comments would be already be an improvement
